packages/calcite-components/src/components/link/link.scss

```diff
@@ -1,3 +1,13 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-link-text-color: Specifies the component's text color.
+ * @prop --calcite-link-border-color: Specifies the component's border color.
```

Comment:
@alisonailea i'm not sure what to call this one. Its an underline so should it be `--calcite-link-underline-color`? 

Its not really a border because its using a linear gradient background.